tests/post/steps/test_dns.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from pytest_bdd import scenario, given, when, then, parsers
+
+# Scenarios
+@scenario('../features/dns_resolution.feature', 'check dns')
+def test_dns(host):
+    pass
+
+@then(parsers.parse("The hostname '{hostname}' should be resolved"))
+def resolve_hostname(host, hostname):
+    with host.sudo():
+        # create a testing deployment
+        cmd = ("kubectl --kubeconfig=/etc/kubernetes/admin.conf"
+               " run --image=busybox busybox"
+               "--generator=run-pod/v1"
```

Comment:
I think a space is missing here. Or if not, then we don't need trailing space at beginning of new command strings.